src/server/lib/encryptedStateCookie.ts

```diff
@@ -35,9 +35,17 @@ export const setEncryptedStateCookie = (
 export const readEncryptedStateCookie = (
   req: Request,
 ): EncryptedState | undefined => {
-  const encryptedCookie = baseUri.includes('localhost')
-    ? req.cookies[encryptedStateCookieName]
-    : req.signedCookies[encryptedStateCookieName];
+  // TODO: We need to double check the security implications of using an env variable
+  // to read from the unsigned cookie storage.
```

Comment:
@coldlink this is the TODO I was talking about, also I noticed this is the only change in the PR that doesn't just affect documentation or tests so we probably want to make sure we are happy with this.

Reply:
Ah the environment variable it's referring to I think is the `RUNNING_IN_CYPRESS` one. Which is fine, as long as we make sure that isn't set on any production environment (which is isn't currently). The encryption secret keys are always stored in either github action or aws parameter store, and we don't use the same keys for any environment.

Reply:
So I think this is fine.